dht_net.go

```diff
@@ -108,41 +112,60 @@ func (dht *IpfsDHT) handleNewMessage(s inet.Stream) {
 	}
 }
 
+func tagWithMessage(ctx context.Context, name string, msg *pb.Message) {
+	log.SetTag(ctx, name+".key", b58.Encode([]byte(msg.GetKey())))
```

Comment:
This is going to be a bit expensive. Let's wait for https://github.com/libp2p/go-libp2p-kad-dht/pull/200, then we can use the `loggableKey` helper type to format lazily and on-demand.